mkimage.sh

```diff
@@ -100,6 +100,12 @@ ENV DEBIAN_FRONTEND noninteractive
 LABEL io.resin.architecture="rpi" \
 	  io.resin.qemu.version="$RESIN_QEMU_VERSION" \\
 	  io.resin.device-type="raspberrypi"
+RUN apt-get update && apt-get install -y --no-install-recommends wget \
+	&& wget http://archive.raspberrypi.org/debian/raspberrypi.gpg.key \
+	&& apt-key add raspberrypi.gpg.key \
+	&& rm -rf raspberrypi.gpg.key \
+	&& echo "deb http://archive.raspberrypi.org/debian/ jessie main" >> /etc/apt/sources.list \
```

Comment:
Is this the debian repo? It doesn't look like it has armv6 packages and mixing debian and raspbian doesn't look like a good idea